packages/static-wado-webserver/lib/middlewares/cors.mjs

```diff
@@ -0,0 +1,16 @@
+import cors from "cors";
+import dicomWebServerConfig from "../dicomWebServerConfig.mjs";
```

Comment:
Don't direct import the configuration here, but instead get it passed in - there is no guarantee that this configuration object will be used at all for this server setup, so you can't just use it.